Library/Homebrew/brew.sh

```diff
@@ -6,7 +6,10 @@ case "$HOMEBREW_SYSTEM" in
 esac
 
 # Force UTF-8 to avoid encoding issues for users with broken locale settings.
-if [[ "$(locale charmap 2>/dev/null)" != "UTF-8" ]]
+if ! command -v locale >/dev/null
+then
+  export LC_ALL=C
+elif [[ "$(locale charmap 2>/dev/null)" != "UTF-8" ]]
```

Comment:
```suggestion
elif [[ "$(locale charmap)" != "UTF-8" ]]
```
I'd prefer not to suppress stderr. We do not expect `locale charmap` to fail.